### PR TITLE
Add "wrong" tag to contacts

### PR DESCRIPTION
**Pull request by KFad11**
Adds a wrong tag to contacts so that we can display them differently than the other contacts.

### CODE REVIEW

File: app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb [hunk 1]
```diff
@@ -15,4 +15,13 @@
           contact_detail: contact[:contact_detail],
       }) %>
     <% end %>
+    <% unless object.wrong_contacts.length == 0 %>
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb]: `unless` makes things murky to others when reading 

```erb
<% if object.wrong_contact.present? %>
```

File: app/pb_kits/playbook/pb_person_contact/person_contact.rb [hunk 2]
```diff
@@ -9,6 +9,17 @@ class PersonContact
...
       prop :people, type: Playbook::Props::HashArray, default: []
+      prop :wrong_contacts, type: Playbook::Props::HashArray, default: []
+
+      def filter_number
+        if contact_type = "wrong number"
+          wrong_number << contact
+          positon = contacts.find_index(contact)
+          contacts.delete_at(position)
+        end
+        wrong_number
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_person_contact/person_contact.rb]: I don't think this does anything right?

File: app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb (hunk 1 above)
Comment (kskilt) [reply app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb]: I tried using `.present?` and it was not actually filtering. `.length == 0` is filtering properly, but I do agree its ugly if there was a cleaner method.

Comment (kskilt) [reply app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb]: So it is passing an array, so we were able to change this to `.empty?` which gave the same functionality and is much cleaner.

**Comment by KFad11** [comment]
Hey @thestephenmarshall - We've got everything good to go for review on this one. Let us know if anything needs to change. We can always take the `contact_detail`s off the docs for now if that is still in limbo with how/if we want to implement that.

**Pull request merged**